storage/modules/block_storage.go

```diff
@@ -956,6 +970,14 @@ func (b *BlockStorage) storeTransaction(
 	blockIdentifier *types.BlockIdentifier,
 	tx *types.Transaction,
 ) error {
+	backwardRelationKeys := b.getBackwardRelationKeys(tx)
+	for _, key := range backwardRelationKeys {
+		err := storeUniqueKey(ctx, transaction, key, []byte{}, true)
```

Comment:
this won't necessarily be unique unfortunately. We probably just want to do a `transaction.Set`